src/UserApp.ts

```diff
@@ -0,0 +1,173 @@
+import { DivsStructure } from "../utils/HtmlStructures"
+import { $, render, toLCaseIncludes, toArr } from "./variables/functions"
+
+
+// Enum with members representing buttons on the top of the app
+export enum CurrentButton {
+    Search,
+    Add,
+    Remove,
+    ShowAll
+}
+// User class
+export class User {
+    name: string
+    surname: string
+    dateOfBirth: number
+
+    constructor( name: string, surname: string, dateOfBirth: number ) {
+        this.name = name
+        this.surname = surname
+        this.dateOfBirth = dateOfBirth
+    }
+
+}
+// State class
+class UserAppState {
+    // Current button type
+    btnState: CurrentButton
+    // Array of found users
+    foundUsers: Array<User> = []
+    // Check if submit button clicked
+    ifSbmtClicked: boolean = false
+    // Check if user's input is right
+    ifExecutedRight: boolean
+    // Array of users
+    users: Array<User> = []
+    // Removed users
+    removedUser = []
+    // TODO here (constructor, function bodies etc)
+    constructor( selectedButton: CurrentButton ) {
+        this.btnState = selectedButton
+    }
+    // Add new User to array
+    addUser( name: string, surname: string, dateOfBirth: number ) {
+        this.users.push( new User( name, surname, dateOfBirth ) )
+        return this.users
+    }
+    // Put eaither found users array either users array to render body 
+    changeUsers( clickedBtn: CurrentButton ) {
```

Comment:
The problem with this function is that it does not change anything, even tho the name suggests it does.

The way how it works, you should call it something like `alternateBetweenUserCollections()`, just to be able to more or less clearly show what this does (returns one collection or another, depending on the current state).